src/core/src/package_managers/YumPackageManager.py

```diff
@@ -583,19 +583,19 @@ def disable_auto_os_update_for_packagekit(self):
         #todo: uncomment after finding the correct value
         # self.update_os_patch_configuration_sub_setting(self.download_updates_identifier_text, "false", self.packagekit_config_pattern_match_text)
         self.update_os_patch_configuration_sub_setting(self.apply_updates_identifier_text, "false", self.packagekit_config_pattern_match_text)
-        self.disable_auto_update_on_reboot(self.dnf_automatic_disable_on_reboot_cmd)
+        self.disable_auto_update_on_reboot(self.packagekit_disable_on_reboot_cmd)
 
-        self.composite_logger.log("Successfully disabled auto OS updates using dnf-automatic")
+        self.composite_logger.log("Successfully disabled auto OS updates using packagekit")
 
     def is_service_set_to_enable_on_reboot(self, command):
         """ Checking if auto update is enable_on_reboot on the machine. An enable_on_reboot service will be activated (if currently inactive) on machine reboot """
         self.composite_logger.log_debug("Checking if auto update service is set to enable on reboot...")
         code, out = self.env_layer.run_command_output(command, False, False)
         self.composite_logger.log_debug(" - Code: " + str(code) + ", Output: \n|\t" + "\n|\t".join(out.splitlines()))
         if len(out.strip()) > 0 and code == 0 and 'enabled' in out:
-            self.composite_logger.log_debug("Auto OS update will enable on reboot")
+            self.composite_logger.log_debug("Auto OS update service will enable on reboot")
```

Comment:
This (and similar) would benefit from disambiguation in the future -- which service? (can be inferred from other lines, but think about reviewing telemetry at scale)

Reply:
Had kept the service to be inferred from other logs around this, but it will be better to have the service name in here if possible. Adding this to Task# 11007199